tmt/utils.py

```diff
@@ -529,6 +529,22 @@ def verbose(
         if not self.opt('quiet') and self.opt('verbose') >= level:
             echo(self._indent(key, value, color, shift), err=err)
 
+    def verbose3(
+            self,
+            key: str,
+            value: Optional[str] = None,
+            color: Optional[str] = None,
+            shift: int = 2,
+            level: int = 3,
+            err: bool = False) -> None:
+        """
+        Show message if in requested verbose mode level (>=3 by default)
+
+        In quiet mode verbose messages are not displayed.
+        """
+        self.verbose(key=key, value=value, color=color, shift=shift,
+                     level=level, err=err)
```

Comment:
Feels a bit ad hoc-ish.

* I'd drop the `level` parameter - if the purpose of this helper is to provide a callable with `level=3` baked in, then offering the parameter does not make sense to me. It would let me call `verbose3(..., level=1, ...)` - please, don't let me do that :)
* I miss other variants, like `verbose1`, `verbose2`, etc. - when I see `verbose3`, I would definitely expect other levels to be materialized too. And there's also debug verbosity one can tweak as well, i.e. `debug*`...
* `shift` default value is changed & hard-coded, too, which seems to me like the biggest issue here, as this particular value of `shift` is tailored for one particular use case. If a plugin developer decides to call `self.verbose3` from their module, they might get an absolutely wrong indentation. `shift` should mirror the default of `verbose()` itself.

I'm not excited about the name & verbosity of its definition, I was hoping `functools.partial` could help us a bit:

```python
    verbose1 = functools.partial(verbose, level=1)
    verbose2 = functools.partial(verbose, level=2)
    verbose3 = functools.partial(verbose, level=3)
    verbose4 = functools.partial(verbose, level=4)
```

But mypy can't handle these shortcuts, and `self.verbose3(11)` is not reported as a mistake (the first argument is supposed to be `str`, `int` should raise a mypy error). I'm afraid adding such helpers would have to be more verbose after all...